### DISCUSSION TITLE
How to use Data Controller?

### DISCUSSION THREAD

**Discussion by Phoen-Polarcube** [Ark2000/PankuConsole#151] [Q&A] [answered]
Hi, I found it difficult to use Data Controller window.
I even can't open the window, Is there any specific command to open the window?
I've tried to select the object through the `interactive shell` but, nothing happened 😔

**Comment by Ark2000** [comment, marked as answer]
see https://github.com/Ark2000/PankuConsole/blob/master/docs/data_controller.md

the object must contain supported export variables, so data controller can be created.
